server/files/javascript/visibility.js

```diff
@@ -62,6 +62,7 @@ semantic.visiblity.ready = function() {
 
   $firstSticky
     .sticky({
+      silent         : true,
```

Comment:
is silent really required?

Reply:
The problem is: When calling visiblity the item are hidden all the time, thus the log warning will always occur.
But, the mentioned necessary refresh is indeed included in visiblity.js already
https://github.com/fomantic/Fomantic-UI-Docs/blob/a1cb52d73df4a8bd6f0de3b5833bdc8de816eff7/server/files/javascript/visibility.js#L177-L182

however it's done/called at a later stage within visibility.html 
https://github.com/fomantic/Fomantic-UI-Docs/blob/a1cb52d73df4a8bd6f0de3b5833bdc8de816eff7/server/documents/behaviors/visibility.html.eco#L770-L772

So, to just get rid of the warning, i added the silent setting.